src/server-manager/cache-manager.ts

```diff
@@ -72,16 +93,16 @@ export class CacheManager {
    *                Instance Methods
    * ************************************************************************
    */
-  private cacheImageMessageRawPayload? : AsyncMap<string, PadplusMessagePayload>
-  private cacheContactRawPayload?     : AsyncMap<string, PadplusContactPayload>
-  private cacheRoomMemberRawPayload?  : AsyncMap<string, {
-    [contactId: string]: PadplusRoomMemberPayload,
+  private cacheImageMessageRawPayload?           : AsyncMap<string, WechatyCacheMessagePayload>
+  private cacheContactRawPayload?                : AsyncMap<string, WechatyCacheContactPayload>
+  private cacheRoomMemberRawPayload?             : AsyncMap<string, {
+    [contactId: string]: WechatyCacheRoomMemberPayload,
```

Comment:
We have declared `xxxRoomMemberMap`, I think it is better to use it.

Reply:
ok.